server/repository/mongo/service_shifts.py

```diff
@@ -87,5 +87,6 @@ def get_shifts(self, shift_ids):
         Returns:
             list: A list of ServiceShift objects.
         """
-        shifts = self.collection.find({"_id": {"$in": shift_ids}})
+        object_ids = [ObjectId(sid) for sid in shift_ids]
+        shifts = list(self.collection.find({"_id": {"$in": object_ids}}))
```

Comment:
@emrameduseljac These changes probably have to do with the changes you did in domains/service_shift.py. Can you explain why they are needed.

The code generally works and tests pass, I'm just not understanding this detail.